docs/general/whatsinside.rst

```diff
@@ -30,6 +30,11 @@ The full `Bootstrap library <http://getbootstrap.com>`_ is imported via the
 `Sass component <https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/private/sass/libs/_bootstrap.scss>`_
 and the `JavaScript component <https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/static/js/libs/bootstrap.min.js>`_.
 
+.. note::
+
+    We use a 24 column based grid system instead of the default 12.
```

Comment:
something more the line "Aldryn Bootstrap 3 uses a 24 column based grid setting instead of the default 12. You can change this setting in..." (if there is somewhere a setting)